app/api/resources/user.py

```diff
@@ -286,8 +292,19 @@ class UserRegister(Resource):
         HTTPStatus.CREATED.value, "%s" % messages.USER_WAS_CREATED_SUCCESSFULLY
     )
     @users_ns.response(
-        HTTPStatus.BAD_REQUEST.value,
-        f"{messages.USERNAME_FIELD_IS_EMPTY}\n{messages.PASSWORD_INPUT_BY_USER_HAS_INVALID_LENGTH}\n{messages.USER_INPUTS_SPACE_IN_PASSWORD}\n{messages.EMAIL_INPUT_BY_USER_IS_INVALID}",
+        HTTPStatus.BAD_REQUEST,
```

Comment:
i think you have to use `.value` otherwise, it will show on Swagger UI as "HTTPStatus.BAD_REQUEST" instead of "400"

Reply:
Done